hack/ci/testdata/kubermatic_dualstack.yaml

```diff
@@ -24,6 +24,8 @@ spec:
   imagePullSecret: '__IMAGE_PULL_SECRET__'
   userCluster:
     apiserverReplicas: 2
+    machineController:
+      imageTag: "32c8ec0998445a664938d2f7783a9d12168adf7b"
```

Comment:
Maybe mark this as TODO? It can be removed later after MC version is updated.

Reply:
Yes. It might just get forgotten tho. 

I will create reminder in the sig-networking for next week. If we have new release then update otherwise snooze one more week. What do you say?

Reply:
yeah week or two :)